Sources/Fullscreen/PopupConsentView/PopupView.swift

```diff
@@ -4,15 +4,16 @@
 
 import UIKit
 
-// MARK: - ConsentViewDelegate
+// MARK: - PopupViewDelegate
 
-public protocol ConsentViewDelegate: NSObjectProtocol {
-    func consentView(_ consentView: ConsentView, didSelectYesButton button: Button)
-    func consentView(_ consentView: ConsentView, didSelectNoButton button: Button)
-    func consentView(_ consentView: ConsentView, didSelectCancelButton button: Button)
+public protocol PopupViewDelegate: NSObjectProtocol {
+    func popupView(_ popupView: PopupView, didSelectBottomRightButton button: Button)
```

Comment:
Yes, as generic as possible. As you see from the screenshots the buttonTitles are different for every case. Therefore, I think it can be confusing to use names like that.